src/app/notification/toast-notification-list.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="toast-notifications-list-pf" *ngIf="notifications && notifications.length > 0">
+  <div *ngFor="let notification of notifications">
+    <alm-toast-notification
```

Comment:
if you don't want to switch to 'pf' yet then use 'f8-' to namespace all the elements